pkg/adapter/adapter.go

```diff
@@ -302,30 +304,52 @@ func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 	}
 }
 
+func setMsgContentType(msg wabbit.Delivery) string {
+	contentType := fmt.Sprint(msg.Headers()["Content-type"])
+	switch contentType {
+	case cloudevent.ApplicationJSON, cloudevent.TextJSON, "":
```

Comment:
I used just the types that were defined on the event library

Reply:
those are just the minimum types, in the sdk you can register anything. We should expect everything and anything to be enqueued.